src/server/resolvers/helpers.test.ts

```diff
@@ -0,0 +1,227 @@
+import { ObjectId, MongoClient } from 'mongodb';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import { toEnum, checkIsAuthorized, query, queryById, fetchUser, updateUser } from './helpers';
+import {
+	ShirtSize,
+	UserType,
+	UserDbInterface,
+	HackerDbObject,
+	OrganizerDbObject,
+} from '../generated/graphql';
+import { initDbWithConnStr, Models } from '../models';
+
+const hacker: UserDbInterface = {
+	_id: new ObjectId(),
+	createdAt: new Date(),
+	dietaryRestrictions: [],
+	email: 'foo@bar.baz',
+	firstName: 'Foo',
+	lastName: 'Bar',
+	logins: [],
+	preferredName: 'Foo',
+	secondaryIds: [],
+	userType: UserType.Hacker,
+};
+
+let mongoServer: MongoMemoryServer;
+let mongoClient: MongoClient;
+let models: Models;
+
+const testHackerId = new ObjectId();
+const testHacker = ({
+	_id: testHackerId,
+	email: 'foo@bar.com',
+} as unknown) as HackerDbObject;
+const testOrganizerId = new ObjectId();
+const testOrganizer = ({
+	_id: testOrganizerId,
+	email: 'foo@bar.com',
+} as unknown) as OrganizerDbObject;
+
+beforeAll(async () => {
+	try {
+		mongoServer = new MongoMemoryServer();
+		const mongoUri = await mongoServer.getConnectionString();
+		models = await initDbWithConnStr(mongoUri);
+		await models.Hackers.insertOne(testHacker);
```

Comment:
make sure `mongod.getInstanceInfo();` is good

Reply:
before the inserts

Reply:
The class will now throw if unsuccessful in the connection.